Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+
+#if !defined(Q_MOC_RUN)
+#include <Editor/Plugins/Cloth/ClothJointWidget.h>
+#include <Editor/Plugins/HitDetection/HitDetectionJointWidget.h>
+#include <UI/PropertyEditor/ReflectedPropertyEditor.hxx>
+#include <EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/DockWidgetPlugin.h>
+#include <Editor/Plugins/SkeletonOutliner/SkeletonOutlinerBus.h>
+#include <AtomToolsFramework/Inspector/InspectorWidget.h>
+#include <Editor/Plugins/Ragdoll/RagdollNodeWidget.h>
+#include <QPushButton>
+#endif
+namespace EMStudio {
```

Comment:
please add an empty line between line 13 and line 14, also between line 17 and 18.